draft/index.html

```diff
@@ -258,6 +258,9 @@
 Nebst einer Kurzbeschreibung wird angegeben, ob der Wert eine Pflichtangabe ist und um welchen Typ es sich handelt.
 Wenn erforderlich wird der Wertebereich festgelegt und es werden kurze Beispiele gegeben.
 
+<section id="allgemein">
```

Comment:
Die ID-Bezeichnungen sollten ggf. einheitlich in Englisch sein. Hier könnte somit "general" oder "common" verwendet werden.